packages/frontend/src/components/Layout/NavigationSidebar.tsx

```diff
@@ -57,6 +58,12 @@ function NavigationSidebarItem({
   )
 }
 
+const templateDrawerLink: DrawerLink = {
+  Icon: BiBookOpen,
+  text: 'Templates',
+  to: URLS.TEMPLATES,
+}
+
```

Comment:
better to move the `Layout/index.tsx`, under drawerLinks, together with other nav routes. messy to have drawerlinks in different files. Add a new optional flag e.g. isBottom to conditionally render them on top or bottom

Reply:
ok thanks ill move this to the `Layout` page instead!